attributes/default.rb

```diff
@@ -39,12 +39,12 @@
 #
 # Kibana
 #
-default['kibana']['version']                    = "7.2.0"
-default['kibana']['url']                        = "#{node['download_url']}/kibana-oss-#{node['kibana']['version']}-linux-x86_64.tar.gz"
+default['kibana']['version']                    = "1.1.0"
+default['kibana']['url']                        = "#{node['download_url']}/opensearch/opensearch-dashboards-#{node['kibana']['version']}-linux-x64.tar.gz"
+#default['kibana']['sha512']                     = "57c3b59b8f5970e781f2ea78db98af9d0b0ff183dcb15f2e87b8ff29098704ab08b093c0bd2dcb05ea01883a0c49572d54800d8789e5398c36b2aa1f56179ba2"
 default['kibana']['port']                       = "5601"
-
 default['kibana']['systemd']                    = "true"
-default['kibana']['home']                       = node['hopslog']['dir'] + "/kibana-" + "#{node['kibana']['version']}-linux-x86_64"
+default['kibana']['home']                       = node['hopslog']['dir'] + "/opensearch-dashboards-#{node['kibana']['version']}"
 default['kibana']['base_dir']                   = node['hopslog']['dir'] + "/kibana"
```

Comment:
shouldn't we change the base_dir to `opensearch-dashboards`?

Reply:
I didn't want to break any dependencies on it, but i guess i should just find any.